### PR TITLE
Release 2.25.0

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]

**Review by beorn7 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Alertmanager API v2 was released in Alertmanager v0.16.0 (released in January
 * [ENHANCEMENT] TSDB: Reload blocks every minute, to detect new blocks and enforce retention more often. #8343
```
Comment (ArthurSens) [inline CHANGELOG.md:20]: Not sure how bad of a problem this is, but this PR increases the probability of race conditions with the clean tombstones API.

#8007 changes the logic of this API to reload several times as well, that's when the race was discovered and later on fixed.

Comment (roidelapluie) [reply CHANGELOG.md:20]: Was the race present before , but only eevery 2 hours? What is the consequence?

Comment (roidelapluie) [reply CHANGELOG.md:20]: Can we have a fix for "just" the race? How hard would that be?

Comment (ArthurSens) [reply CHANGELOG.md:20]: Yes, the chance of race was almost impossible before. One would have to call the api exactly when a reload was happening, which was once every 2h.

Now that we reload every minute, if the reload takes too much time, which is unlikely,then we have a chance of race

Comment (ArthurSens) [reply CHANGELOG.md:20]: The consequence is the possibility of Prometheus trying to delete a block twice. A block can be cleaned by retention limits and also by the CleanTombstones API. I think it's fine for linux, but on windows that could crash Prometheus

Comment (codesome) [reply CHANGELOG.md:20]: There is no race. CleanTombstones() and every minute reloadBlocks() are protected by db.cmtx :)

**Review by codesome — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ArthurSens) [reply CHANGELOG.md:20]: cc @codesome. Do you think I'm right with the comments above?

Comment (ArthurSens) [reply CHANGELOG.md:20]: Okey, I saw my mistake now. Sorry for making you folks worry unnecessarily

**Pull request merged**